sailon/random_novelty_detector.py

```diff
@@ -10,10 +8,55 @@
 
 from smqtk_core import Configurable, Pluggable
```

Comment:
Aren't these imports unused now?

Reply:
@waxlamp @Pikus16 

Should I be using `Plugfigurable` instead of `Configurable` and `Pluggable`? I can't find `Plugfigurable` in `smqtk` or `smqtk_core`.

Reply:
Not really, since you are inheriting from `ONDAdapter` so you don't have to import either of these as far as I can see. Thus the imports are unused

Reply:
My mistake, you're correct. I thought this comment was referencing the file in which `ONDAdapter` was defined. These imports have now been removed.